packages/core-data/src/actions.js

```diff
@@ -209,29 +214,73 @@ export function* redo() {
  * @param {string} kind    Kind of the received entity.
  * @param {string} name    Name of the received entity.
  * @param {Object} record  Record to be saved.
+ * @param {Object} options Saving options.
  */
-export function* saveEntityRecord( kind, name, record ) {
+export function* saveEntityRecord(
+	kind,
+	name,
+	record,
+	{ isAutosave = false } = { isAutosave: false }
+) {
 	const entities = yield getKindEntities( kind );
 	const entity = find( entities, { kind, name } );
 	if ( ! entity ) {
 		return;
 	}
-	const key = entity.key || DEFAULT_ENTITY_KEY;
-	const recordId = record[ key ];
+	const entityIdKey = entity.key || DEFAULT_ENTITY_KEY;
+	const recordId = record[ entityIdKey ];
 
-	yield { type: 'SAVE_ENTITY_RECORD_START', kind, name, recordId };
+	yield { type: 'SAVE_ENTITY_RECORD_START', kind, name, recordId, isAutosave };
 	let error;
 	try {
-		const updatedRecord = yield apiFetch( {
-			path: `${ entity.baseURL }${ recordId ? '/' + recordId : '' }`,
-			method: recordId ? 'PUT' : 'POST',
-			data: record,
-		} );
-		yield receiveEntityRecords( kind, name, updatedRecord, undefined, true );
+		const path = `${ entity.baseURL }${ recordId ? '/' + recordId : '' }`;
+		if ( isAutosave ) {
+			const persistedRecord = yield select(
+				'getEntityRecord',
+				kind,
+				name,
+				recordId
+			);
+			const currentUser = yield select( 'getCurrentUser' );
+			const currentUserId = currentUser ? currentUser.id : undefined;
+			const autosavePost = yield select(
+				'getAutosave',
+				persistedRecord.type,
+				persistedRecord.id,
+				currentUserId
+			);
+			let data = { ...persistedRecord, ...autosavePost, ...record };
+			data = Object.keys( data ).reduce( ( acc, key ) => {
+				if ( key in [ 'title', 'excerpt', 'content' ] ) {
```

Comment:
Is this something to be configured per entity (autosaveable fields or something)

Reply:
Yeah, good idea.